Makefile

```diff
@@ -54,21 +52,6 @@ driver_uninstall:
 	@rm -fv $(DESTDIR)/$(MODULEDIR)/razerfirefly.ko
 	@rm -fv $(DESTDIR)/$(MODULEDIR)/razercore.ko
 
-
-# Launchpad hacks
-lp_driver:
-	@echo -e "\n::\033[32m Compiling OpenRazer kernel modules (Launchpad)\033[0m"
-	@echo "========================================"
-	$(eval KERNELDIR:=/lib/modules/$(shell dpkg --get-selections | grep -P 'linux-headers-.+(generic|amd64)' | awk '{print $$1}' | tail -1 | sed 's/linux-headers-//g')/build)
-	make -C $(KERNELDIR) SUBDIRS=$(DRIVERDIR) modules
-
-lp_driver_clean:
-	@echo -e "\n::\033[32m Cleaning OpenRazer kernel modules (Launchpad)\033[0m"
-	@echo "========================================"
-	$(eval KERNELDIR:=/lib/modules/$(shell dpkg --get-selections | grep -P 'linux-headers-.+(generic|amd64)' | awk '{print $$1}' | tail -1 | sed 's/linux-headers-//g')/build)
-	make -C "$(KERNELDIR)" SUBDIRS="$(DRIVERDIR)" clean
```

Comment:
These `lp_` targets are still used in the Launchpad recipes (so for Ubuntu packages)

Reply:
The `lp_` targets are not used anymore since https://github.com/openrazer/openrazer/commit/f09aa4a8da3698c38806a836da5eba139f93fb40 , the current debian/rules file doesn 't contain any reference to `lp_` targets. So, we can safely remove them, or I miss something?

Reply:
I'd say it's safe to remove. There isn't any trace nor any errors on the [openrazer-daily](https://code.launchpad.net/~openrazer/+recipe/openrazer-daily) recipe since it's removal.

Reply:
Remove them then :)